Rewrite community values (#5411)
kubernetes/community: values.md

# Kubernetes Community Values

> The culture of Kubernetes contributes substantially to the project's success.Below are the distilled values which have evolved over the years in our community, pushing our project and peers toward constant improvement.

Reviewer: The original `values.md` used two spaces after each period although one space is also common.
I suggest two spaces between "success.Below".

## Distribution is better than centralization

The scale of the Kubernetes project is only viable through high-trust and high-visibility distribution of work, which includes delegation of authority, decision making, technical design, code ownership, and documentation.  Distributed asynchronous ownership, collaboration, communication and decision making are the cornerstones of our world-wide community.

## Community over product or company

> We are here as a community first, our allegiance is to the intentional stewardship of the Kubernetes project for the benefit of all its members and users everywhere.  We support working together publicly for the common goal of a vibrant interoperable ecosystem, providing an excellent experience for our users. Individuals gain status through work, companies gain status through their commitments to support this community and fund the resources necessary for the project  to operate.

Reviewer: There is two spaces between "project  to", I suggest to using one space.

## Automation over process

Large projects have a lot of less exciting, yet, hard work.  We value time spent automating repetitive work more highly than toil. Where the work cannot be automated, it is our culture to recognize and reward all types of contributions. However, heroism is not sustainable.

## Inclusive is better than exclusive

> Broadly successful and useful technologies requires different perspectives and skill sets, which can only be heard in a welcoming and respectful environment.  Community membership is a privilege, not a right. Community Leadership is earned through effort, scope, quality, quantity, and duration of contributions. Our community shows respect for the time and effort put into a discussion, regardless of where a contributor is on their growth path.

Reviewer: ```suggestion
Broadly successful and useful technologies require different perspectives and skill sets, which can only be heard in a welcoming and respectful environment.  Community membership is a privilege, not a right.  Community Leadership is earned through effort, scope, quality, quantity, and duration of contributions.  Our community shows respect for the time and effort put into a discussion, regardless of where a contributor is on their growth path.
```

Reviewer: If `values.md` continues to use two spaces after a period then I suggest to have the spacing consistent as suggested.

The suggested change of "technologies requires" to "technologies require" is from the [#sig-docs Slack conversation](https://kubernetes.slack.com/archives/C1J0BPD2M/p1610662908010300?thread_ts=1610651040.009400&cid=C1J0BPD2M) from Slack user @adamc.

Reviewer 2: I should have just posted my suggestions here. I just was being a bit lazy on Slack! Thanks for mentioning it here.

# Kubernetes Community Values

Kubernetes Community culture contributes substantially to the project's success. The following values have evolved over time, pushing our project and peers toward constant improvement.

## Distribution is better than centralization

> The scale of the Kubernetes project is only viable through high-trust and high-visibility distribution of work, which includes delegation of authority, decision making, technical design, code ownership, and documentation.  Distributed asynchronous ownership, collaboration, communication and decision making are the cornerstones of our world-wide community.

Reviewer 3: non-blocking nit: I agree the verb and pluralization should agree, but would "is the cornerstone" read better? I bumped on the plural "cornerstones".

I _think_ "Distributed $stuff is the cornerstone..." is still coherent, but I'll defer to the docs folks.

Reviewer 4: @liggitt 

> would "is the cornerstone" read better?

For what it's worth, I think neither of them read especially well. :shrug: I'd like to use different language altogether, but I wanted to scope both the originating issue and any implementation reviews conservatively.

For purposes of this PR, I think either is/singular or are/plural can work.

# Kubernetes Community Values

> The culture of Kubernetes contributes substantially to the project's success.  Below are the distilled values which have evolved over the years in our community, pushing our project and peers toward constant improvement.

Reviewer 5: I believe it is important to retain the reference to Kubernetes _Community_ with regards to culture. The second part could be put into active voice: The values below are distilled from years of community evolution and drive constant improvement in the project.

Reviewer 4: "are distilled from" still uses passive voice. Maybe something like:
```suggestion
Kubernetes Community culture contributes substantially to the project's success. The following values evolved over years of community evolution, pushing our project and peers toward constant improvement.
```

## Distribution is better than centralization

The scale of the Kubernetes project is only viable through high-trust and high-visibility distribution of work, which includes delegation of authority, decision making, technical design, code ownership, and documentation.  Distributed asynchronous ownership, collaboration, communication and decision making are the cornerstones of our world-wide community.

## Community over product or company

> We are here as a community first, our allegiance is to the intentional stewardship of the Kubernetes project for the benefit of all its members and users everywhere.  We support working together publicly for the common goal of a vibrant interoperable ecosystem, providing an excellent experience for our users.  Individuals gain status through work, companies gain status through their commitments to support this community and fund the resources necessary for the project to operate.

Reviewer 5: It's a nit, but single spaces after periods is now the standard. This is a habit I have to break in my own writing as well.

Reviewer 4: ```suggestion
We are here as a community first. Our allegiance is to the intentional stewardship of the Kubernetes project for the benefit of all its members and users everywhere. We support working together publicly for the common goal of a vibrant interoperable ecosystem, providing an excellent experience for our users. Individuals gain status through work. Companies gain status through their commitments to support this community and fund the resources necessary for the project to operate.
```

## Automation over process

> Large projects have a lot of less exciting, yet, hard work.  We value time spent automating repetitive work more highly than toil. Where the work cannot be automated, it is our culture to recognize and reward all types of contributions. However, heroism is not sustainable.

Reviewer 5: I think we can omit "that" or leave it. Changing it to "the" makes it generic whereas the original phrasing is in reference to the prior sentence. I lean towards omitting it.

Author: Yes done

> Large projects have a lot of less exciting, yet, hard work.  We value time spent automating repetitive work more highly than toil. Where the work cannot be automated, it is our culture to recognize and reward all types of contributions. However, heroism is not sustainable.

Reviewer 4: ```suggestion
Large projects have a lot of hard yet less exciting work. We value time spent automating repetitive work more highly than toil. Where work cannot be automated, our culture recognizes and rewards all types of contributions while recognizing that heroism is not sustainable.
```

## Inclusive is better than exclusive

> Broadly successful and useful technologies require different perspectives and skill sets, which can only be heard in a welcoming and respectful environment.  Community membership is a privilege, not a right.  Community Leadership is earned through effort, scope, quality, quantity, and duration of contributions.  Our community shows respect for the time and effort put into a discussion, regardless of where a contributor is on their growth path.

Reviewer 5: Single space after period is stylistically correct.

Reviewer 4: ```suggestion
Broadly successful and useful technologies require different perspectives and skill sets, which can only be heard in a welcoming and respectful environment. Community membership is a privilege, not a right. Community members earn leadership through effort, scope, quality, quantity, and duration of contributions. Our community respects the time and effort put into a discussion, regardless of where a contributor is on their growth path.
```

## Evolution is better than stagnation

> Openness to new ideas and studied technological evolution make Kubernetes a stronger project.  Continual improvement, servant leadership, mentorship and respect are the foundations of the Kubernetes project culture. It is the duty for leaders in the Kubernetes community to find, sponsor, and promote new community members. Leaders should expect to step aside. Community members should expect to step up.

Reviewer 4: ```suggestion
Openness to new ideas and studied technological evolution make Kubernetes a stronger project. Continual improvement, servant leadership, mentorship, and respect are the foundations of Kubernetes culture. Kubernetes community leaders have a duty to find, sponsor, and promote new community members. Leaders should expect to step aside. Community members should expect to step up.
```
